services/src/integ-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceIntTest.java

```diff
@@ -264,21 +247,7 @@ public void testConcurrentDuplicateSubmissions() throws IOException {
             Assert.assertNotNull(request.getRequestPayload());
         }
 
-        try {
-            // Call the bulk submission service for second time without clearing the concurrencyMap from the first call.
-            bulkSubmissionService.saveBulkSubmission(bulkSubmission);
-
-            Assert.fail("Should have thrown exception");
-        } catch (final Throwable e) {
-            if (!(e instanceof CustomerReferenceNotUniqueException) ||
-                    !e.getMessage().matches(
-                            "Failed with code \\[DUP_CUST_FILEID\\]; "
-                                    + "message\\[Duplicate User File Reference 10711 supplied. This was "
-                                    + "previously used to submit a Bulk Request on .*? and "
-                                    + "the SDT Bulk Reference \\S*? was allocated.\\]")) {
-                Assert.fail("Unexpected exception returned\n" + e.getStackTrace());
-            }
-        }
+        assertThrows(CustomerReferenceNotUniqueException.class, () -> bulkSubmissionService.saveBulkSubmission(bulkSubmission));
```

Comment:
Although more compact this is no longer checking the message contained within the exception as the original version did.  Test should check the exception message.

Reply:
need to discuss about this